components/homepage/Hero.tsx

```diff
@@ -5,13 +5,16 @@ type Props = {};
 
 const Hero = (props: Props) => {
   return (
+
     <div className=" 2xl:mx-auto 2xl:max-w-[120rem] mx-auto flex w-full max-w-[30rem] flex-col justify-center bg-white1 pb-[2rem] pt-[3rem] sm:mx-auto sm:max-w-[30rem] md:mx-auto md:max-w-[60rem] md:flex-row lg:mx-auto lg:max-w-[75rem] xl:mx-auto xl:max-w-[90rem]">
       <div className="flex flex-col md:min-w-[32rem]">
         <h4 className="title mb-[1.25rem] md:mb-[1.69rem]">HI, I AM AUSTIN</h4>
         <h1 className="text-[2.25rem] md:text-[4rem]">
           Professional
           <br />
           Web Developer
+          {/* z index negative */}
+          <div className="z-[-1] h-[1rem] w-full bg-yellow"></div>
```

Comment:
```suggestion
<div className="underline-magic">Based in USA</div>
```
Let's make this expandable. You'll need to provide some edits and perhaps change some things on use-case. But this is how I would approach writing a line behind some text:

Consider adding something like this to your css:
```css
  .underline-magic {
    position: relative !important;
    z-index: 1 !important;
  }
  .underline-magic::after {
    content: "";
    position: absolute;
    top: 50%;
    left: 0;
    right: 0;
    height: 20px;
    background-color: rgb(78, 248, 0);
    z-index: -1;
  }
  ```
  
  
![image](https://github.com/austinmargarone/portfolio/assets/4108484/32e814ef-e217-4321-94b1-1193355ab358)